### PR TITLE
fix: add missing dependencies for build

### PR DESCRIPTION
**Pull request by Bakkenrak**
# Add missing dependencies

## Description
I wasn't able to run the npm build script locally nor in the CI workflow. Unless I'm missing something, these dependencies just need to be added.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

Build no longer fails due to missing dependencies

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by cremich** [comment]
Thanks @Bakkenrak
I am interested to know what kind of error you get when running a local build without these deps. 
Could you paste it here?

**Comment by Bakkenrak** [comment]
Yeah, I was figuring that this behavior and need for explicit referencing was strange. It's actually not occuring on my machine anymore.
Before the error was the same as in this workflow test run: https://github.com/Bakkenrak/kiko-backend/runs/2374301317?check_suite_focus=true
It's strange though that it happened there as well.

**Comment by cremich** [comment]
I looked the deps and saw that codecommit is missing. Maybe this is the root cause for the build errors. 

@Bakkenrak could you try to verify if you still get errors after you added `@aws-cdk/aws-codecommit`

**Comment by Bakkenrak** [comment]
It does seem to fix part of the dependency errors in the pipeline. But there are still a bunch left and a TS error that I actually also get locally.

**Comment by cremich** [comment]
What kind of cdk have you installed locally and what kind of tsc errors do you get? Let us maybe create an issue for this. Maybe others have the same problem.

File: package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
   "dependencies": {
     "@aws-cdk/aws-amplify": "^1.98.0",
     "@aws-cdk/aws-appsync": "^1.98.0",
+    "@aws-cdk/aws-codecommit": "^1.99.0",
```
Comment (cremich) [inline package.json:38]: Could you use version 1.98.0 instead? We will upgrade to 1.99.0 soon.

**Review by cremich — CHANGES_REQUESTED** [review]
Please change to version 1.98.0

File: package.json (hunk 1 above)
Comment (Bakkenrak) [reply package.json]: Ah right, that was an oversight. Now the remaining dependency errors are gone.

**Review by cremich — APPROVED** [review]

**Pull request merged**